backend/src/main/java/ai/giskard/web/dto/TestSuiteCreateDTO.java

```diff
@@ -6,17 +6,19 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import java.util.UUID;
+
 @Getter
 @Setter
 @NoArgsConstructor
 @UIModel
 public class TestSuiteCreateDTO {
     private Long projectId;
```

Comment:
Generally speaking Long are good values for being entity ids, they're even better than UUIDs since they don't require as much space when you store them. 
I changed Models' and Datasets' ids to UUID because they're objects created on the client side and uploaded to Giskard. In this case we can't use incremental ids to avoid the following situation:
- you connect ML Worker to your server A and execute a test that requires a model with id 10
- ML Worker downloads this model from the server
- ML Worker runs the test
- You connect your ML Worker to another server and run a test from there. By coincidence this server also has a model with id 10, but it's not the same model as on  server A. But since ML worker already has a model 10 in its cache it won't re-download it again and will try to run the test using this model (will likely crash)

with UUIDs you don't have such a problem because all ids are unique*

Reply:
But following the same logic, won't we have a case where an ML worker connects to another server B that by coincidence has the same projectId as the server A but for a different project?

Reply:
The chances are lower since project keys aren't incremental and set by users. Also, even in a project with the same key exists and you try to load a dataset or a model with project key and UUID it won't be found.

Reply:
Okay fair enough.